### PR TITLE
Some styling fixes

### PR DESCRIPTION
**Pull request by ctmcquilkin**
The form looks better, still a work in progress.

### CODE REVIEW

File: source/frontend/css/main.scss [hunk 1]
```diff
@@ -162,310 +162,389 @@ Main components
...
 
-
 /* -------------------------------- 
 
 Form
 
 -------------------------------- */
-.ca-form {
+
+.cd-form {
```
Comment (paulrenenichols) [inline source/frontend/css/main.scss]: @ctmcquilkin Did you mean *cd-form* or *ca-form* here?

File: source/frontend/css/main.scss [hunk 2]
```diff
@@ -162,310 +162,389 @@ Main components
...
+    color: darken($color-3, 30%);
     margin-bottom: 10px;
   }
-  .ca-form input[type="text"],
-  .ca-form input[type="password"],
-  .ca-form input[type="email"],
-  .ca-form textarea,
-  .ca-form select {
+
+  .cd-label {
```
Comment (paulrenenichols) [inline source/frontend/css/main.scss]: @ctmcquilkin Is *cd-label* what you intended?

**Comment by ctmcquilkin** [comment]
Fixed title of login form, added icons, found/replaced all "cd" instances.

**Comment by paulrenenichols** [comment]
Looks good, @ctmcquilkin.  I'm going to merge this, but I found this warning in the output of the CSS build:

> WARNING: The box-sizing mixin is deprecated and will be removed in the next major version release. This property can now be used un-prefixed.
>  on line 410 of source/frontend/vendor/css/bourbon/_bourbon-deprecated-upcoming.scss
>  from line 18 of stdin

**Comment by ctmcquilkin** [comment]
No problem. I can remove that mixin tomorrow or later this week.

**Pull request merged**